alacritty/src/renderer/platform.rs

```diff
@@ -69,6 +71,23 @@ pub fn pick_gl_config(
         };
 
         if let Some(gl_config) = gl_config {
+            if log::max_level() >= LevelFilter::Debug {
+                let mut msg = String::from("Picked GL config:");
+                let _ = write!(msg, "\n  buffer_type: {:?}", gl_config.color_buffer_type());
+                let _ = write!(msg, "\n  alpha_size: {:?}", gl_config.alpha_size());
+                let _ = write!(msg, "\n  num_samples: {:?}", gl_config.num_samples());
+                let _ =
+                    write!(msg, "\n  hardware_accelerated: {:?}", gl_config.hardware_accelerated());
+                let _ = write!(msg, "\n  config_api: {:?}", gl_config.api());
+                let _ = write!(
+                    msg,
+                    "\n  supports_transparency: {:?}",
+                    gl_config.supports_transparency()
+                );
+                let _ = write!(msg, "\n  srgb_capable: {:?}", gl_config.srgb_capable());
+                debug!("{}", msg);
+            }
```

Comment:
This seems like the worst possible way to write this. Wouldn't it be easier to just use `format!`?

Reply:
I think I've found a worse alternative.